libs/api/domains/icelandic-names-registry/src/lib/services/backend.ts

```diff
@@ -0,0 +1,53 @@
+import { Inject, Injectable } from '@nestjs/common'
+import { DataSourceConfig } from 'apollo-datasource'
+import { RESTDataSource } from 'apollo-datasource-rest'
+
+import {
```

Comment:
The naming convention for services seems to include `.service.ts` postfix, consider updating this to maintain consistency